Runtime/GameLiftServerProcessConfiguration.cs

```diff
@@ -1,13 +1,14 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-namespace AmazonGameLift.Runtime
+namespace AmazonGameLiftPlugin.Core
 {
     [System.Serializable]
-    public struct GameLiftConfiguration
+    public struct GameLiftServerProcessConfiguration
```

Comment:
Typically GameLift is synonymous with a GameLift Server, so this rename may not be required. Thoughts?

Reply:
For backwards compatibility, I think changing this might cause a lot of customer pain. I have discovered a bunch of calls to it within the Sample game, so we would have expected the customers to also do the same.